test/integration/targets/git/tasks/archive.yml

```diff
@@ -74,3 +74,63 @@
   when:
     - "ansible_os_family == 'RedHat'"
     - ansible_distribution_major_version is version('7', '>=')
+
+- name: ARCHIVE | Inspect archive file
+  command:
+    cmd: "{{ git_list_commands[item] }} {{ checkout_dir }}/test_role.{{ item }}"
+    warn: no
+  register: archive_content
+  with_items: "{{ git_archive_extensions[ansible_os_family ~ ansible_distribution_major_version | default('default') ] | default(git_archive_extensions.default) }}"
+
+# Does not work on RedHat6 (jinja2 too old)
+- name: ARCHIVE | Ensure archive content is correct
+  assert:
+    that:
+      - item.stdout_lines | sort | first == 'defaults/'
+  with_items: "{{ archive_content.results }}"
+  when:
+    - "ansible_os_family == 'RedHat'"
+    - ansible_distribution_major_version is version('7', '>=')
```

Comment:
This will skip Ubuntu and OpenSUSE. Try only skipping on RedHat6:
```
 ansible_os_family ~ ansible_distribution_major_version != 'RedHat6'
```

Reply:
BTW, other tests in [test/integration/targets/git/tasks/archive.yml](../tree/devel/test/integration/targets/git/tasks/archive.yml) have the same condition since I don't know when. Should I replace them with `ansible_os_family ~ ansible_distribution_major_version != 'RedHat6'` too?